src/pages/setup.tsx

```diff
@@ -19,7 +20,12 @@ const DynamicForm = dynamic(() => import('@/components/ui/form/Form'), {
 
 export default function Home() {
   const [community] = useRecoilState(communityState)
-  const { address, isConnected } = useAccount()
+  const { address } = useAccount()
+  const [showConnect, setShowConnect] = React.useState(false)
+
+  React.useEffect(() => {
```

Comment:
@kristoferlund `useEffect` is not needed here, a simple const with a boolean assertion of `address` is enough. We should rarely use `useEffect`, check [this section of the new React docs as a reference](https://react.dev/learn/you-might-not-need-an-effect)

Reply:
For reference, this is why I wrapped `address` in `useEffect`. 

https://nextjs.org/docs/messages/react-hydration-error